src/toil/cwl/cwltoil.py

```diff
@@ -591,13 +591,19 @@ def _abs(self, path: str) -> str:
 
         Overwrites cwltool.stdfsaccess.StdFsAccess._abs() to account for toil specific schema.
         """
-        
+
         # Used to fetch a path to determine if a file exists in the inherited cwltool.stdfsaccess.StdFsAccess,
         # (among other things) so this should not error on missing files.
         # See: https://github.com/common-workflow-language/cwltool/blob/beab66d649dd3ee82a013322a5e830875e8556ba/cwltool/stdfsaccess.py#L43
         if path.startswith("toilfs:"):
```

Comment:
`readGlobalFile` is supposed to throw `NoSuchFileException` if the file you *ask for* doesn't exist, and is *never* supposed to return a nonexistent path. So if it is doing so (which it was, until I fixed it), we can't necessarily rely on it throwing when it would.

I think if I move it out of the Toil side so it also covers the side that delegates, we'll break the `exists()` implementation, which expects to be able to call `_abs()` with nonexistent paths while only needing to catch `NoSuchFileException`.

I think I want to leave it as is and fix the spelling. If `readGlobalFile` does anything other than throw `NoSuchFileException` or return a path that exists, it is doing crimes, but `_abs()` in general is allowed to be called on nonexistent files (by `exists()`). So being called on one shouldn't be a failing assert.